# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/nricectx.h
# Commit: f5353ea9f1df
# Full Hash: f5353ea9f1dfe15a65076bc7aa272d688ebdb8ca
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Add a proxy policy argument to NrSocketProxyConfig, and always set this config on the NrIceCtx. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45101
# ==============================================================================

diff -r ed1ae6a85255 -r f5353ea9f1df media/mtransport/nricectx.h
--- a/media/mtransport/nricectx.h	Tue Sep 17 18:17:13 2019 +0000
+++ b/media/mtransport/nricectx.h	Tue Sep 17 18:17:14 2019 +0000
@@ -306,12 +306,10 @@
     return proxy_config_;
   }
 
-  void SetCtxFlags(bool default_route_only, bool proxy_only);
-
-  bool proxy_only() const { return proxy_only_; }
+  void SetCtxFlags(bool default_route_only);
 
   // Start ICE gathering
-  nsresult StartGathering(bool default_route_only, bool proxy_only,
+  nsresult StartGathering(bool default_route_only,
                           bool obfuscate_host_addresses);
 
   // Start checking
@@ -395,7 +393,6 @@
   Policy policy_;
   RefPtr<TestNat> nat_;
   std::shared_ptr<NrSocketProxyConfig> proxy_config_;
-  bool proxy_only_;
   bool obfuscate_host_addresses_;
   std::map<std::string, std::string> obfuscated_host_addresses_;
 };